roles/prometheus/server/tasks/install.yml

```diff
@@ -6,6 +6,51 @@
     name: cloudalchemy.prometheus
     tasks_from: preflight
 
+- name: Upload {{ prometheus_dist_file }}
```

Comment:
I added a sentence in the README, I don't really know what to say, don't hesitate if you have another proposal.

Reply:
It's normal for getting started, that's why we have a `stable` branch and a `latest` one. It will be necessary to make a PR in the getting started just after this one to add the download of the archive.

I added in the README the link where to find the releases for Prometheus.